src/views/InvolvementsAll/index.jsx

```diff
@@ -173,87 +170,117 @@ const InvolvementsAll = () => {
                   </Select>
                 </FormControl>
               </Grid>
-              <Grid item xs={12} md={6} lg={3}>
-                <FormControl fullWidth variant="filled">
-                  <InputLabel id="activity-type">Type</InputLabel>
-                  <Select
-                    labelId="activity-type"
-                    id="activity-type"
-                    value={type}
-                    onChange={(event) => setType(event.target.value)}
-                  >
-                    <MenuItem label="All" value="">
-                      <em>All</em>
-                    </MenuItem>
-                    {types.map((type) => (
-                      <MenuItem value={type} key={type}>
-                        {type}
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-              </Grid>
             </Grid>
           </CardContent>
```

Comment:
I think that, rather than ending here, this `CardContent` element should wrap the entire `TabContext` as well (and you should remove any `CardContent`s within the `TabPanel`s below).